debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/ShowDumpAgentsCommand.java

```diff
@@ -113,6 +115,8 @@ public ShowDumpAgentsCommand()
 		/* Known dump events */
 //		{ "gpf",         "ON_GP_FAULT",             J9RAS_DUMP_ON_GP_FAULT },
 		rasDumpEventsToNames.put(J9RAS_DUMP_ON_GP_FAULT, "gpf");
+//		{ "user2",       "ON_USER2_SIGNAL",         J9RAS_DUMP_ON_USER2_SIGNAL },
+		rasDumpEventsToNames.put(J9RAS_DUMP_ON_USER2_SIGNAL,"user2");
```

Comment:
I think `J9RAS_DUMP_ON_USER2_SIGNAL` is always non-zero (see `runtime/oti/j9dump.h`) since it's the identifier of the event. I don't think this should be conditional on a compilation flag since I think one `jdmpview` compiled without the `user2` event could be used to process a core dump produced by a VM compiled with the `user2` event.

Reply:
In older core files, `J9RAS_DUMP_ON_USER2_SIGNAL` won't exist, but the flag should not be set so including this unconditionally should be safe.